packages/api-server/src/plugins/lambdaLoader.ts

```diff
@@ -40,17 +40,17 @@ export const setLambdaFunctions = async (foundFunctions: string[]) => {
         )
       }
       // TODO: Use terminal link.
-      console.log(
-        c.magenta('/' + routeName),
-        c.dim.italic(Date.now() - ts + ' ms')
-      )
+      console.log(chalk.magenta('/' + routeName))
+      console.log(chalk.dim.italic(Date.now() - ts + ' ms'))
```

Comment:
This is now two lines instead of all being logged on a single line.
Just commenting to make sure you made this change on purpose

Reply:
Ah thanks, I probably didn't want that